JustFakeIt/MatchingStrategy/BodyMatching/PartialJsonMatching.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace JustFakeIt
+{
+    public class PartialJsonMatching : IBodyMatchingOption
+    {
+        public bool MatchBody(string expectedBody, string actualBody)
+        {
+            return string.IsNullOrEmpty(expectedBody) || actualBody.Equals(expectedBody) || JsonKeyValuesMatch(expectedBody, actualBody);
+        }
+
+        public bool JsonKeyValuesMatch(string expectedBody, string actualBody)
+        {
+            JObject expectedJObject;
+            JObject actualJObject;
+
+            try
+            {
+                expectedJObject = JsonConvert.DeserializeObject<JObject>(expectedBody);
```

Comment:
Do you need to specify `JObject` here? What does `JsonConvert.DeserializeObject(expectedBody)` give you?

Also, what happens if `expectedBody` is null? I guess you'd get an exception, but you don't explicitly check for null here.

Don't care that much, though.